packages/webdriverio/package.json

```diff
@@ -78,19 +80,22 @@
     "aria-query": "^5.0.0",
     "css-shorthand-properties": "^1.1.1",
     "css-value": "^0.0.1",
-    "devtools": "8.12.1",
-    "devtools-protocol": "^0.0.1170846",
+    "decamelize": "^6.0.0",
```

Comment:
aren't we missing the chromedriver package?

Reply:
The `chromedriver` NPM package installs Chromedriver on install rather then runtime which would be less ideal for this scenario (all other drivers are able to dynamically detect the browser and install the right driver). The Chrome team has worked on an NPM package (`@puppeteer/browser`) that allows dynamic installing Chromedriver and Chrome.